packages/workflow/src/Expression.ts

```diff
@@ -277,7 +277,16 @@ export class Expression {
 					throw error;
 				}
 			}
+
+			if (
+				error instanceof Error &&
+				typeof error.message === 'string' &&
+				error.name === 'SyntaxError'
+			) {
+				throw new Error(error.message);
```

Comment:
Why is this check needed if it's throwing same error with same message?

Reply:
To surface the `SyntaxError`, preventing it from becoming `null`.